packages/web-components/case-card/src/element.ts

```diff
@@ -74,32 +64,17 @@ export class DenhaagCaseCardElement extends HTMLElement implements CardData {
           <div class="denhaag-card__background"></div>
           <div class="denhaag-card__content">
             <div class="denhaag-card__text-wrapper">
-              <p class="utrecht-paragraph denhaag-card__title">
+              <div class="denhaag-card__title">
                 <slot name="heading"></slot>
-              </p>
+              </div>
               ${subtitleSlot}
+              <slot></slot>
             </div>
             <div class="denhaag-card__actions">
               ${this.getDateTimeElement()}
               <a aria-label=${this.linkLabel} href="${escapeXML(this.href)}" class="denhaag-card__action-link"
```

Comment:
Kwam dit net tegen, hier gaat de aria-label nog fout, de double quotes missen:

`aria-label=${this.linkLabel}` => `aria-label="${this.linkLabel}`

Gevolg is dat de href in de aria-label terecht komt, zie ook de href in de codepen: https://codepen.io/bddjong/pen/LEEYREY